examples/arkane/species/C2H5/ethyl.py

```diff
@@ -19,14 +19,15 @@
     'Klip_2': -78.98344186,
 }
 
-geometry = Log('ethyl_cbsqb3.log')
+geometry = Log('ethyl_b3lyp.log')
 
-frequencies = Log('ethyl_cbsqb3.log')
+frequencies = Log('ethyl_b3lyp.log')
 
 """pivot are the two atoms that are attached to the rotor
 top contains the atoms that are being rotated including one of the atoms from pivots
 symmetry is the symmetry number of the scan
-fit is fit of the scan data. It defaults to 'best', but can also be assigned as 'cosine' or 'fourier'"""
-rotors = [
-    HinderedRotor(scanLog=Log('ethyl_scan_72.log'), pivots=[1,2], top=[1,3,4], symmetry=6, fit='best')
-]
```

Comment:
Is it worth doing the same thing to `examples/arkane/species/Toluene_Hindered_Rotor`?

Reply:
I wanted to leave one example in with the explicit symmetry number (although it's well documented) for users who would like to specify it. But I'm open to other thoughts.